sys/crypto/chacha.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2008  D. J. Bernstein  (dedicated to the public domain)
+ * Copyright (C) 2015  René Kijewski  <rene.kijewski@fu-berlin.de>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
```

Comment:
There are a lot of trailing whitespaces in this file.
Check https://travis-ci.org/RIOT-OS/RIOT/jobs/67709906 for more details